buildSrc/src/main/kotlin/VersionUtils.kt

```diff
@@ -1,3 +1,10 @@
+import org.gradle.api.Project
+
+fun Project.getVersionName(): String? =
+    findProperty("jellyfin.version")?.toString()
+        ?: System.getenv("JELLYFIN_VERSION")
+        ?: "0.0.0-dev.1"
```

Comment:
Shouldn't the tag also work in local development environments? That way, it'd be easier to quickly switch between release variants and local builds. Also, I find the `0.0.0` a bit weird :thinking:

Or is the `jellyfin.version` designed as a method to override the version name locally?

Reply:
The function first looks using Gradle's `findProperty` function if a version is set in the properties files. If it can't find any it will look for an environment variable (this one is set in the Azure pipeline) and if it can't find that it will default to `0.0.0-dev.1` which will be parsed as version code `1`.

We might want to add a separate package-id for the debug build like we did with ATV too so you can use both release and debug at the same time.

Reply:
That sounds reasonable. I'll merge this then.